doc/en/user/source/community/geofence/cache.rst

```diff
@@ -0,0 +1,48 @@
+.. _geofence_cache:
+
+GeoFence Cache REST
+===================
+
+The Geofence client cache can be queried and invalidated through a REST service.
+
+Requests
+--------
+
+``/geofence/ruleCache/info``
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Retrieve information about the geofence caches.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 10 20 20 20
+
+   * - Method
+     - Action
+     - Parameters
+     - Response
+   * - GET
+     - Retrieve information about the geofence caches  (such as cache size, hits, misses, success, failures, load times and evictions)
```

Comment:
cache singular? What is the difference between hits/misses and success/failures ? If you want to keep them all, it probably needs to be "successes" to match the other plural forms.

Reply:
I am assuming that cache hits and misses refers to finding the object in the cache, or not. (load) successes and failures refers to loading the object when there is a cache miss. I'll add the word "load" to successes and failures